.github/workflows/build-plugin-archive.yml

```
@@ -202,16 +215,27 @@ jobs:
           composer global require inpsyde/wp-translation-downloader
           composer wp-translation-downloader:download
 
+      - name: Run PHP Scoper
+        if: steps.composer-tools.outputs.php-scoper == '0'
+        run: |
+          php-scoper add-prefix --force --output-dir build
```

Can we align the flag syntax?
```suggestion
          php-scoper add-prefix --force --output-dir=build
```